substreams-helper/src/common.rs

```diff
@@ -61,3 +61,9 @@ impl HasAddresser for StoreGetInt64 {
         self.get_last(key.to_hex()).is_some()
     }
 }
+
+impl HasAddresser for Address {
```

Comment:
Is this just impled for Addresses only? If so perhaps you could change the store args below to have name address rather than store and type as Address rather than impl Store to make it more clear what is compared with what here?

Reply:
It is also implemented for all substream stores. So if you have a Store with addresses of contracts it works too.

Reply:
Didn't realise. Ok makes sense, thanks 👍